recipes/repository.rb

```diff
@@ -10,13 +10,16 @@
   when 'apt'
     include_recipe 'apt::default'
 
+    apt_key = 'CBCB082A1BB943DB'
+    apt_key = 'F1656F24C74CD1D8' if node['platform'] == 'ubuntu' && node['platform_version'].split('.')[0].to_i >= 16
```

Comment:
It's documented [here](https://mariadb.com/kb/en/mariadb/installing-mariadb-deb-files/#import-the-gnupg-signing-key) on the MariaDB.

I just noticed they mention Debian too but the tests pass so, shrug:

> As of Ubuntu 16.04 "Xenial" and Debian Sid, our signing key has changed. The id of the new signing key is 0xF1656F24C74CD1D8 ...

If it ain't broke don't fix it?  Seriously though, this PR is just to add tests.  I only fixed the open issue about the new key in order for tests to pass.